resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -282,7 +284,7 @@ install.packages('IRkernel')
 IRkernel::installspec()
 ```
 
-To see if you were successful, try running Jupyter Lab and seeing if you have working R kernel. To launch the Jupyter Lab type the following in the Windows Powershell:
+To see if you were successful, try running Jupyter Lab and check if you have working R kernel. To launch the Jupyter Lab type the following in the Windows Powershell:
```

Comment:
Why does this say Windows Powershell? Should it not be terminal?

Reply:
Not sure where powershell came from, fixed.